controllers/operands/kubevirt_test.go

```diff
@@ -2524,6 +2527,90 @@ Version: 1.2.3`)
 			}))
 		})
 
+		Context("Tune KubeVirt rateLimiters", func() {
+
+			var hco *hcov1beta1.HyperConverged
+
+			BeforeEach(func() {
+				hco = commonTestUtils.NewHco()
+			})
+
+			It("Should be empty by default", func() {
+				kv, err := NewKubeVirt(hco)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(hco.Spec.Tuning).To(BeNil())
+				Expect(kv.Spec.Configuration.APIConfiguration).To(BeNil())
+				Expect(kv.Spec.Configuration.ControllerConfiguration).To(BeNil())
+				Expect(kv.Spec.Configuration.WebhookConfiguration).To(BeNil())
+				Expect(kv.Spec.Configuration.HandlerConfiguration).To(BeNil())
+
+			})
+
+			It("Should be empty if configmap is not present", func() {
+				hco.Spec.Tuning = &hcov1beta1.Tuning{
+					Policy: &hcov1beta1.Policy{
+						Name: "static-config"},
+				}
+
+				kv, err := NewKubeVirt(hco)
+				Expect(err).To(HaveOccurred())
+				Expect(kv).To(BeNil())
+
+			})
+
+			It("Should be empty if the configmap is present but the parameters are wrong", func() {
+				hco.Spec.Tuning = &hcov1beta1.Tuning{
+					Policy: &hcov1beta1.Policy{
+						Name: "static-config"},
+				}
+				//burst is missing
+				configMapData := make(map[string]string, 0)
+				configMapData["qps"] = "100"
+
+				configMap := corev1.ConfigMap{
+					TypeMeta: metav1.TypeMeta{
+						Kind:       "ConfigMap",
+						APIVersion: "v1",
+					},
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "static-config",
+						Namespace: getNamespace(hco.Namespace, []string{}),
+					},
+					Data: configMapData,
+				}
+
+				//WIP: not able to create a mock cluster for testing
+				resources := []runtime.Object{}
+				cl := commonTestUtils.InitClient(resources)
+
+				ci := commonTestUtils.ClusterInfoMock{}
+				Expect(ci.IsOpenshift()).To(BeTrue())
+
+				mgr, err := commonTestUtils.NewManagerMock(&rest.Config{}, manager.Options{}, cl, logger)
+				Expect(err).ToNot(HaveOccurred())
+				mockmgr, _ := mgr.(*commonTestUtils.ManagerMock)
+
+				//	config, err := cfg.GetConfig()
+				//	Expect(err).ToNot(HaveOccurred())
+				clientSet, err := kubernetes.NewForConfig(mockmgr.GetConfig())
+				//	Expect(err).ToNot(HaveOccurred())
+
+				mockmgr.GetClient()
+				_, err = clientSet.CoreV1().ConfigMaps(hco.Namespace).Create(context.TODO(), &configMap, metav1.CreateOptions{})
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
The easiest way to create the config map is to create a CM object and add it to the resources; e.g.
```suggestion
				//WIP: not able to create a mock cluster for testing
				resources := []runtime.Object{configMap}
				cl := commonTestUtils.InitClient(resources)
```